src/wallet/rpc/backup.cpp

```diff
@@ -1987,6 +2001,28 @@ RPCHelpMan getwalletpakinfo()
     const auto& desc = Parse(desc_str, provider, error);
 
     ret.pushKV("bitcoin_descriptor", desc_str);
+    {
+        CPubKey masterpub = pwallet->offline_xpub.pubkey;
+        secp256k1_pubkey masterpub_secp;
+        int secp256k1_ret = secp256k1_ec_pubkey_parse(secp256k1_ctx, &masterpub_secp, masterpub.begin(), masterpub.size());
+        if (secp256k1_ret != 1) {
+            throw JSONRPCError(RPC_WALLET_ERROR, "bitcoin_descriptor could not be parsed.");
+        }
+
+
+        // Negate the pubkey
+        secp256k1_ret = secp256k1_ec_pubkey_negate(secp256k1_ctx, &masterpub_secp);
+
+        std::vector<unsigned char> negatedpubkeybytes;
+        negatedpubkeybytes.resize(33);
```

Comment:
I think I'd prefer this as a 33 byte array and skip the vec resize, but that's minor